### ISSUE TITLE
Copy invoice

### ISSUE THREAD

**Issue by DebJanson** [default-settings/invoicr#7]
Facility to copy invoice and also to mail merge to multiple recipients

**Comment by harryspink** [comment]
@DebJanson - you can now input a comma separated list of invoices. Example, 'harry@invoicr.io, harry@sandbox.media, harry@defaultsettings.co.uk' would send a copy of the email to all the 3 email addresses.

**Comment by harryspink** [comment]
@DebJanson - Is that what you meant by MailMerge? or do you mean to send 1 email with multiple invoices attached?

**Comment by DebJanson** [comment]
yes - that's what I meant - kind of. but also, the ability to (e.g.) generate the same template that sends to lots of different people but is personalised for each in terms of contact details and reference. Not urgent!
